rasa/core/constants.py

```diff
@@ -20,21 +20,35 @@
 
 # the lowest priority intended to be used by machine learning policies
 DEFAULT_POLICY_PRIORITY = 1
+
+# The priority of intent-prediction policies.
+# This should be below all rule based policies but higher than ML
+# based policies. This enables a loop inside ensemble where if none
+# of the rule based policies predict an action and intent prediction
+# policy predicts one, it's prediction is chosen by the ensemble and
+# then the ML based policies are again run to get the prediction for
+# an actual action. To prevent an infinite loop, intent prediction
+# policies only predict an action if the last event in
+# the tracker is of type `UserUttered`. Hence, they make at most
+# one action prediction in each conversation turn. This allows other
+# policies to predict a winning action prediction.
+UNLIKELY_INTENT_POLICY_PRIORITY = 2
```

Comment:
How come these aren't defined relative to the other constants?
```
DEFAULT_POLICY_PRIORITY = 1
UNLIKELY_INTENT_POLICY_PRIORITY = DEFAULT_POLICY_PRIORITY + 1
```

Reply:
(and so on) I think it might help make things clearer, though I realise this is just my opinion.

Reply:
That's a very good point and I don't know why is it so but it makes sense to me too. 😅

Reply:
Changed it.